vision/unit_tests/test_module.py

```diff
@@ -15,7 +15,9 @@
 from vision.module.in_frame import ModuleInFrame as mif
 from vision.module.location import ModuleLocation
 from vision.module.region_of_interest import region_of_interest
-from vision.module.module_orientation import get_module_orientation
+from vision.module.module_orientation import *
+from vision.module.module_bounding import getModuleBounds
+from vision.module.get_module_depth import *
```

Comment:
Should specify the specific modules you are importing. Using apostrophes is unclear.

Unrelated:
Also, there was a big refactor that got pulled into develop, so getModuleBounds is now get_module_bounds. Will need to change that throughout your benchmark here to reflect that. Same thing with getCenter being changed to get_center, which needs to be changed throughout this file.
The benchmark was also refactored, which is what is causing a merge conflict.